app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
+    <string name="leave_type">Hình thức xin phép</string>
+    <string name="compensation_time">Đăng ký thời gian làm bù</string>
+    <string name="example_time_from">Ví dụ: 2017/05/20 17:30</string>
+    <string name="example_time_to">Ví dụ: 2017/05/20 18:30</string>
+    <string name="confirm_request_leave">Xác nhận tạo yêu cầu đi trễ, về sớm</string>
+    <string name="submit">Gửi đi</string>
+    <string name="next">Tiếp theo</string>
+    <string name="status">Trạng thái</string>
+    <string name="all">Tất cả</string>
+    <string name="month">Tháng</string>
+    <string name="date">Ngày:</string>
+    <string name="date_of_creation">Ngày tạo:</string>
+    <string name="day_session">Buổi</string>
+    <string name="time_request">Thời gian yêu cầu:</string>
+    <string name="request_ot">Yêu cầu làm ngoài giờ</string>
+    <string name="confirm_request_overtime">Xác nhận yêu cầu làm ngoài giờ</string>
+    <string name="request_off">Yêu cầu xin nghỉ</string>
+    <string name="confirm_request_off">Xác nhận xin nghỉ</string>
+    <string name="note">Lưu ý: \u0020</string>
+    <string name="am_pm">Buổi sáng: am - am;  Buổi chiều: pm - pm;  Cả ngày: am - pm</string>
+    <string name="off_have_salary_from">Nghỉ có lương từ</string>
+    <string name="off_have_salary_to">Nghỉ có lương đến</string>
+    <string name="off_no_salary_from">Nghỉ không lương từ</string>
+    <string name="off_no_salary_to">Nghỉ không lương đến</string>
+    <string name="off_have_salary_insurance_coverage">Nghỉ có lương - Bảo hiểm trả</string>
+    <string name="edit">Sửa</string>
+    <string name="delete">Xóa</string>
+    <string name="being_handled_by">Nhóm đang xử lý</string>
+    <string name="confirm_edit_request_off">Xác nhận sửa yêu cầu xin nghỉ</string>
+    <string name="confirm_edit_request_overtime">Xác nhận sửa yêu cầu làm ngoài giờ</string>
+    <string name="edit_request_off">Sửa yêu cầu xin nghỉ</string>
+    <string name="edit_profile">Chỉnh sửa thông tin cá nhân</string>
+    <string name="change_password">Đổi mật khẩu</string>
+    <string name="default_group_and_workspace">Nhóm và workspace mặc định</string>
+    <string name="default_group">Nhóm mặc định</string>
+    <string name="default_workspace">Workspace mặc định</string>
+    <string name="notification_settings">Thiết lập thông báo</string>
+    <string name="receive_notification">Nhận thông báo khi có thao tác (bật/tắt TẤT CẢ)</string>
+    <string name="notification_list">Thông báo danh sách nhân viên đặc biệt</string>
+    <string name="notification_Change_workspace">Thông báo thay đổi Workspace</string>
+    <string name="notification_Change_group">Thông báo thay đổi nhóm</string>
+    <string name="update">Cập nhật</string>
+    <string name="receive_notification_or_email">Nhận thông báo hoặc email</string>
+    <string name="edit_request_overtime">Sửa yêu cầu làm ngoài giờ</string>
+    <string name="type_leave">Hình thức xin phép:</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.type_leave` appears to be unused
 Unused resources make applications larger and slow down builds.